druid-shell/src/platform/gtk/window.rs

```diff
@@ -780,7 +812,22 @@ impl WindowState {
     /// Queues a call to `prepare_paint` and `paint`, but without marking any region for
     /// invalidation.
     fn request_anim_frame(&self) {
-        self.window.queue_draw();
+        if self.in_draw.get() {
+            self.request_animation.set(true);
+        } else {
+            let vbox = &self
+                .window
+                .get_children()
+                .first()
+                .unwrap()
+                .clone()
+                .downcast::<gtk::Box>()
+                .unwrap();
+            let first_child = vbox.get_children().last().unwrap().clone();
```

Comment:
Is this also due to GTK4 -> GTK3, or required?

Reply:
well this is all new code. I do now see that this is way to complicated, we store the drawingarea in the state so we can just use that. So I assume that this comment is no longer relevant